### PR TITLE
Add upload feature functionality that sends videos/images for model training

### PR DESCRIPTION
**Pull request by chc5**

### CODE REVIEW

**Comment by chc5** [comment]
Waiting on @MicLi01  to finish his part.

**Comment by chc5** [comment]
ready

File: src/cv/retrieve_model.sh [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
+# Install yolo-coco model
+cd yolo-coco
+curl -O https://raw.githubusercontent.com/pjreddie/darknet/master/cfg/yolov3.cfg 
 curl -O https://raw.githubusercontent.com/pjreddie/darknet/master/data/coco.names
 curl -O https://pjreddie.com/media/files/yolov3.weights
+cd ../
+
+# Install unknown dataset into frame_extractions and run initalize_state
+source main.env
+wget --load-cookies /tmp/cookies.txt "https://docs.google.com/uc?export=download&confirm=$(wget --quiet --save-cookies /tmp/cookies.txt --keep-session-cookies --no-check-certificate 'https://docs.google.com/uc?export=download&id=1ufydwhMYtOhxgQuHs9SjERnkX0fXxorO' -O- | sed -rn 's/.*confirm=([0-9A-Za-z_]+).*/\1\n/p')&id=1ufydwhMYtOhxgQuHs9SjERnkX0fXxorO" -O unknown.rar && rm -rf /tmp/cookies.txt
```
[the remark this answers is not in the document]
Comment (chc5) [reply src/cv/retrieve_model.sh:12]: Gets the unknown dataset from Google Drive, which requires users to have cookies before downloading the dataset

[the remark this answers is not in the document]
Comment (melvinheks) [reply src/cv/retrieve_model.sh:12]: lit

File: src/cv/settings.py [hunk 2]
```diff
@@ -21,6 +21,12 @@
 RECOGNIZER = WORKDIR + os.environ.get('RECOGNIZER_PATH')
 LABEL_ENCODER = WORKDIR + os.environ.get('LABEL_ENCODER_PATH')
 TRACK_TIME = 600
+"""
+  Directories
+"""
+UPLOAD_FOLDER_PATH = '/tmp'
+ALLOWED_EXTENSIONS = {'txt', 'pdf', 'png', 'jpg', 'jpeg', 'gif'}
```
Comment (melvinheks) [inline src/cv/settings.py:28]: I don't see this used

File: src/cv/retrieve_model.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (chc5) [reply src/cv/retrieve_model.sh:12]: very lit

File: src/web/Dockerfile [hunk 3]
```diff
@@ -7,7 +7,8 @@ ENV NODE_ENV=production
 ENV SESSION_SECRET=${SESSION_SECRET:-whateversession}
 ENV CASSANDRA_CLUSTER_IPS=${CASSANDRA_CLUSTER_IPS:-192.168.1.10}
 ENV CASSANDRA_CLUSTER_PORT=${CASSANDRA_CLUSTER_PORT:-9043}
-ENV CASSANDRA_FLASK_SERVER_URL=${CASSANDRA_FLASK_SERVER_URL:-http://192.168.1.10:5001}
+ENV CASSANDRA_FLASK_SERVICE_URL=${CASSANDRA_FLASK_SERVER_URL:-http://192.168.1.10:5001}
```
Comment (yliu0715) [inline src/web/Dockerfile:10]: How come the default URL for this and the one in src/web/server/app/utils/settings.ts is different?

File: src/cv/settings.py (hunk 2 above)
Comment (chc5) [reply src/cv/settings.py]: It'll be an upcoming pr to restrict the file types on the server

Comment (chc5) [reply src/cv/settings.py]: I removed it for now

**Comment by yliu0715** [comment]
yes

File: src/web/Dockerfile (hunk 3 above)
Comment (chc5) [reply src/web/Dockerfile:10]: Opps. I'll take note of that in the next pull request. (I felt like server isn't the right word for that service.)

**Pull request merged**